docs/smtchecker.rst

```diff
@@ -632,7 +632,8 @@ option ``--model-checker-solvers {all,cvc4,eld,smtlib2,z3}`` or the JSON option
 .. note::
   z3 version 4.8.16 broke ABI compatibility with previous versions and cannot
   be used with solc <=0.8.13. If you are using z3 >=4.8.16 please use solc
-  >=0.8.14.
+  >=0.8.14, and vice versa. We also recommend using the latest z3 release which
```

Comment:
This vice versa reads like "if you are using solc >=0.8.14, please use z3 >= 4.8.16" to me, shouldn't it be?

```suggestion
  >=0.8.14, and conversely, use older z3 with older solc releases. We also recommend using the latest z3 release which
```